### PR TITLE
chore: changelog for 1.22

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

Diff:
https://github.com/appium/appium/blob/v1.21.0/npm-shrinkwrap.json
https://github.com/appium/appium/blob/releases/1.22/npm-shrinkwrap.json

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,46 @@
...
+### iOS General
+
+* Xcode 13 and iOS 15 support
+
+### iOS(XCUITest)
+
+
+### Mac2
+* Add `macos:` extensions:
+  - Add support [swift guestures](https://github.com/appium/appium-mac2-driver#mobile-swipe) [appium-mac2-driver#85](https://github.com/appium/appium-mac2-driver/pull/85)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:27]: typo here (`swift`)?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,46 @@
...
+- Add `language` option in `macos: appleScript` to support other languages supported by `osascript` [appium-mac2-driver#89](https://github.com/appium/appium-mac2-driver/pull/89)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:28]: to be able to use other languages supported by osascipt

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,46 @@
...
+- Add support of touch bar interaction [appium-mac2-driver#87](https://github.com/appium/appium-mac2-driver/pull/87)
+- Fix handling libxml2 client to prevent memroy leaks [appium-mac2-driver#86](https://github.com/appium/appium-mac2-driver/pull/86)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:30]: memory leaks
handling of

File: CHANGELOG.md [hunk 4]
```diff
@@ -23,18 +23,24 @@ Appium 1.x will have patch releases, but it will no longer get minor releases.
...
+    * `appium:forceAppLaunch` to launch the application under test forcefully with `appium:bundleId`. Defaults to `true` unless `noReset` capability is set to `true`. [appium-xcuitest-driver#1314](https://github.com/appium/appium-xcuitest-driver/pull/1314)
+* Add  `durationSeconds` argument in `mobile: pressButton` for tvOS
+* Add `getProperty` command to support [Get Element Property](https://www.w3.org/TR/webdriver/#dfn-get-element-property) endpoint. In native context, the command is equal to [Get Element Attribute](https://www.w3.org/TR/webdriver/#get-element-attribute) [appium-xcuitest-driver#1313](https://github.com/appium/appium-xcuitest-driver/pull/1313)
 
 ### Mac2
 * Add `macos:` extensions:
-  * Add support [swipe guestures](https://github.com/appium/appium-mac2-driver#mobile-swipe) [appium-mac2-driver#85](https://github.com/appium/appium-mac2-driver/pull/85)
+    * Add support [swipe guestures](https://github.com/appium/appium-mac2-driver#mobile-swipe) [appium-mac2-driver#85](https://github.com/appium/appium-mac2-driver/pull/85)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:36]: extra spaces here?

File: CHANGELOG.md [hunk 5]
```diff
@@ -23,18 +23,24 @@ Appium 1.x will have patch releases, but it will no longer get minor releases.
 * Xcode 13 and iOS 15 support
 
 ### iOS(XCUITest)
-
+* Add capabilities:
+    * `appium:useNativeCachingStrategy` to reduce a potential stale element error. Please read the capability in [XCUITest capabilities](https://github.com/appium/appium-xcuitest-driver#webdriveragent). Defaults to `true` [appium-xcuitest-driver#1317](https://github.com/appium/appium-xcuitest-driver/pull/1317)
+* Add `mobile:` extension:
+    * `appium:configureLocalization` to change localization settings on the currently booted simulator. Please read [configurelocalization](https://github.com/appium/appium-xcuitest-driver#mobile-configurelocalization) for more details [appium-xcuitest-driver#1300](https://github.com/appium/appium-xcuitest-driver/pull/1300)
+    * `appium:forceAppLaunch` to launch the application under test forcefully with `appium:bundleId`. Defaults to `true` unless `noReset` capability is set to `true`. [appium-xcuitest-driver#1314](https://github.com/appium/appium-xcuitest-driver/pull/1314)
+* Add  `durationSeconds` argument in `mobile: pressButton` for tvOS
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:31]: extra space here

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,60 @@
...
+Appium 1.22.0 is a minor release
+
+This release is the **finial** release for Appium 1.x seriese.
+We're going to move to Appium 2.x [TODO: link to documentation].
+Appium 1.x will have patch releases, but it will no longer get minor releases.
+
+### General
+
+- Add `imageMatchMethod` to be able to customize image mathcing method in [find element by image](http://appium.io/docs/en/advanced-concepts/image-elements/) [appium-base-driver#480](https://github.com/appium/appium-base-driver/pull/480) [appium-support#222](https://github.com/appium/appium-support/pull/222)
+- Add [Release Actions](https://www.w3.org/TR/webdriver1/#dfn-release-actions) endpoint [appium-base-driver#501](https://github.com/appium/appium-base-driver/pull/501)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:13]: what we've added is just a stub. The driver does not support the Release action

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,69 @@
...
+We're going to move to Appium 2.x [TODO: link to documentation].
+Appium 1.x will have patch releases, but it will no longer get minor releases.
+
+### General
+
+* Add `imageMatchMethod` to be able to customize image mathcing method in [find element by image](http://appium.io/docs/en/advanced-concepts/image-elements/) [appium-base-driver#480](https://github.com/appium/appium-base-driver/pull/480) [appium-support#222](https://github.com/appium/appium-support/pull/222)
+* `appium:enableWebviewDetailsCollection` capability defaults to `true` [appium-android-driver#699](https://github.com/appium/appium-android-driver/pull/699) (Thanks @mwakizaka)
+* Support `satellites` option in `setGeoLocation` command as [geo command on emulators](https://developer.android.com/studio/run/emulator-console) [appium-adb#571](https://github.com/appium/appium-adb/pull/571)
+
+### Android General
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:16]: it also makes sense to mention the signing optimization: https://github.com/appium/appium-adb/pull/572

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,76 @@
...
+    * `mobile:configureLocalization` to change localization settings on the currently booted simulator. Please read [configurelocalization](https://github.com/appium/appium-xcuitest-driver#mobile-configurelocalization) for more details [appium-xcuitest-driver#1300](https://github.com/appium/appium-xcuitest-driver/pull/1300)
+* Add `durationSeconds` argument in `mobile: pressButton` for tvOS
+* Add `getProperty` command to support [Get Element Property](https://www.w3.org/TR/webdriver/#dfn-get-element-property) endpoint. In native context, the command is equal to [Get Element Attribute](https://www.w3.org/TR/webdriver/#get-element-attribute) [appium-xcuitest-driver#1313](https://github.com/appium/appium-xcuitest-driver/pull/1313)
+* Update parsing logic to parse `:wdc:` element id by Safari Web inspector in WebView context [appium-xcuitest-driver#1324](https://github.com/appium/appium-xcuitest-driver/pull/1324) [appium#15230](https://github.com/appium/appium/issues/15230)
+    * Probably the host macOS should have Safari 14.1
+* Improve alert checking logic in Web contect [appium-xcuitest-driver#1320](https://github.com/appium/appium-xcuitest-driver/pull/1320)
```
Comment (mwakizaka) [inline CHANGELOG.md:44]: `contect` -> `context`?

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,76 @@
+CHANGES IN VERSION 1.22.0 (FROM 1.21.0)
+===================================
+
+Appium 1.22.0 is a minor release
+
+This release is the **finial** release for Appium 1.x seriese.
+We're going to move to Appium 2.x [TODO: link to documentation].
```
Comment (KazuCocoa) [inline CHANGELOG.md:7]: https://github.com/appium/appium/blob/2.0/docs/en/about-appium/1.x-eol.md

Comment (mykola-mokhnach) [reply CHANGELOG.md:7]: not sure about it. I assume we'd still need to do releases if XCTest introduces a breaking update or a major bug is found until final deprecation

Comment (KazuCocoa) [reply CHANGELOG.md:7]: ah, good point

Comment (KazuCocoa) [reply CHANGELOG.md]: Tweaked the description -> https://github.com/appium/appium/pull/15640/commits/d9c67f726ca1d9514b2dbbd98e064f49562a94de

**Review by sakibguy — COMMENTED** [review]
ok

File: CHANGELOG.md [hunk 10]
```diff
@@ -52,7 +52,7 @@ Appium 1.x will have minor or patch releases only when Appium 1.x gets breaking
     * This fix may improve page source results in some environment
 * Fix visibility handling in `mobile:scroll` with `toVisible` argument [WebDriverAgent#528](https://github.com/appium/WebDriverAgent/pull/528) (Thanks @Dan-Maor)
 * Fix `velocity` argument in `mobile:swipe` [appium-xcuitest-driver#1329](https://github.com/appium/appium-xcuitest-driver/pull/1329)
-
+* Fix: `customSSLCert` capability logic [appium-xcuitest-driver#1334](https://github.com/appium/appium-xcuitest-driver/pull/1334)
```
Comment (sakibguy) [inline CHANGELOG.md:55]: ok

**Review by sakibguy — COMMENTED** [review]
ok

File: CHANGELOG.md (hunk 10 above)
Comment (mykola-mokhnach) [reply CHANGELOG.md:55]: @sakibguy Please stop spamming pool requests. Otherwise we'd have to block you from accessing the appium repository.

**Comment by KazuCocoa** [comment]
- [ ] asyncbox 2.9.2
- [ ] espresso driver (compose)?

**Comment by mykola-mokhnach** [comment]
I don't think we should add compose to release notes (yet). The implementation is very limited and is in alpha stage, e.g. requires a lot more functionality to be implemented and tested. Also, zero documentation yet.

**Comment by KazuCocoa** [comment]
I see. I have no objection to it. > compose
asyncbox is functionality ok (appium 2.0 and 1.x beta already has it since a couple of weeks ago) , but I found redundant dependency https://github.com/jlipps/asyncbox/pull/15. After the publishment and bump the version, I think 1.22 is now ready for iOS 15 release.

**Pull request merged**